app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,5 @@
+class PostsController < ApplicationController
+  def index; end
+
+  def show; end
```

Comment:
- [ ] You are missing the accompanying post show view: `app/views/posts/show.html.erb`. Kindly add the file as well

Reply:
This issue is well resolved! ✔️